roles/bigip_user/tasks/test-post-version-12-specific.yaml

```diff
@@ -17,23 +19,45 @@
       that:
           - result|changed
 
+# This will fail as update_password is on_create by default.
 - name: Create user on a different partition - Idempotent check
   bigip_user:
       server: "{{ inventory_hostname }}"
       server_port: "{{ bigip_port }}"
       user: "{{ bigip_username }}"
       password: "{{ bigip_password }}"
       validate_certs: "{{ validate_certs }}"
+      partition_access:
+        - "Common:guest"
       username_credential: "{{ username_credential }}"
       password_credential: "{{ password_credential }}"
       partition: "{{ partition }}"
+  ignore_errors: true
   register: result
 
 - name: Assert Create user on a different partition - Idempotent check
   assert:
       that:
           - not result|changed
 
+- name: Update password for user on a different partition
+  bigip_user:
+      server: "{{ inventory_hostname }}"
+      server_port: "{{ bigip_port }}"
+      user: "{{ bigip_username }}"
+      password: "{{ bigip_password }}"
+      validate_certs: "{{ validate_certs }}"
+      username_credential: "{{ username_credential }}"
+      password_credential: "{{ password_credential }}"
+      partition: "{{ partition }}"
+      update_password: "always"
+  register: result
+
+- name: Assert Update password for user on a different partition
+  assert:
+      that:
+          - result|changed
+
 - name: Remove user on different partition
   bigip_user:
       server: "{{ inventory_hostname }}"
```

Comment:
somewhere in the functional test file, add the following tests

* users on different partitions. there was a bug in pre-13 that didnt allow exists() to work for this example
* users specifying multiple different partition_access for different partitions
* users specifying multiple different partition_access for same partitions

for example

partition: bar
partition_access:
    - Common:operator
    - bar:user_manager


partition_access:
    - Common:operator
    - Common:user_manager


partition_access:
    - Common:operator
    - bar:resource_admin


i expect some of the above to fail, so make sure the failures coming from bigip are correct.